types/node-expat/index.d.ts

```diff
@@ -22,7 +22,7 @@ export class Parser extends Stream {
 
     parse(buf: Buffer | string, isFinal?: boolean): boolean;
 
-    setEncoding(encoding: string): void;
+    setEncoding(BufferEncoding: string): void;
```

Comment:
Isn't it
``` encoding: BufferEncoding ```

Reply:
Yup, used a regex that was too greedy :)